src/content/docs/infrastructure/host-integrations/host-integrations-list/lighttpd-integration.mdx

```diff
@@ -131,7 +131,7 @@ integrations:
     config:
       name: test
       apis:
-        - event_type: &ltTHE_PLACE_WHERE_METRICS_ARE_CAPTURED&gt
+        - event_type: THE_PLACE_WHERE_METRICS_ARE_CAPTURED
```

Comment:
Could you please keep it into the arrow bracket  like <THE_PLACE_WHERE_METRICS_ARE_CAPTURED>  ?